sale_partner_shipping_default_warehouse/models/res_partner.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    warehouse_id = fields.Many2one(
```

Comment:
To reduce the chances of field name conflicts. Please add help text.
```suggestion
    default_sale_warehouse_id = fields.Many2one(
```